ui/oopsyraidsy/oopsyraidsy_config.ts

```diff
@@ -235,14 +235,14 @@ const templateOptions: OptionsTemplate = {
       return;
     const triggers = savedConfig['triggers'];
 
-    if (!triggers || typeof triggers !== 'object' || Array.isArray(triggers))
+    if (triggers === undefined || typeof triggers !== 'object' || Array.isArray(triggers))
```

Comment:
```suggestion
    if (typeof triggers !== 'object' || Array.isArray(triggers))
```

Reply:
I think this was meant to be `!== null`, because:
![image](https://user-images.githubusercontent.com/6119598/168199024-7ba30905-b336-40b1-9599-a38f7acf47ea.png)
Does that make sense instead?

Reply:
What about to add helper functions like `isObject` and `isNullish` so you don't need to repeat so many times. Or consider to import lodash-es (so that webpack would do the tree-shaking things).

Reply:
> I think this was meant to be !== null

I think `triggers` cannot be null, fwiw.

> What about to add helper functions like isObject and isNullish so you don't need to repeat so many times

Unless I am misunderstanding, a lot of these checks are here for TypeScript.  In other words, if you remove the `Array.isArray` check here, then TypeScript will complain.  If you call a function that says `isObject` instead, that isn't enough information for TypeScript to deduce that it's not an array.

Reply:
In addition to what quisquous said above, the end goal of these PRs is to remove nullish/falsey checks and instead be explicit about what values are being checked, to avoid issues like #4447, or the change being discussed here: https://github.com/quisquous/cactbot/pull/4450#discussion_r871907682, so it makes no sense to just move the nullish/falsey checks to elsewhere and have the underlying potential for bugs continue.